code/game/objects/micro_structures.dm

```diff
@@ -0,0 +1,201 @@
+/obj/structure/micro_tunnel
+	name = "mouse hole"
+	desc = "A tiny little hole... where does it go?"
+	icon = 'icons/obj/structures/micro_structures.dmi'
+	icon_state = "mouse_hole"
+
+	anchored = TRUE
+	density = FALSE
+
+	var/max_accepted_scale = 0.3
```

Comment:
I'd suggest maybe a bit higher. Quite a few of people playing micro use higher size despite being almost minimal RP size because of sprite visibility. 0.5 might do.